#general params
verbose: true
camera_angle: 33.93
update_key_using_det: true
add_first_lan: true 
first_lan_x: 1.42
first_lan_y: -0.028
first_lan_z: 0.15
#keyframe
keyframe_delta_trans: 0.5
keyframe_delta_angle: 0.5
keyframe_delta_time: 1.0
#plane segmentation
num_point_seg: 100
norm_point_thres: 1000
planar_area: 0
#data association related
maha_dist_thres: 1.5
use_maha_dist: false
use_eq_dist: true
eq_dist_thres: 1.5
land_noise_low: 0.4 
land_noise_high: 0.4
#odom information cal
use_const_inf_matrix: true
const_stddev_x: 0.00667
const_stddev_q: 0.00001
#graph slam related
save_graph: false
save_graph_path: /home/hriday/Desktop/semantic_graph.g2o
compute_txt_for_ate: true

